thinkhazard/static/less/common.less

```diff
@@ -194,12 +192,11 @@ ul.horizontal li {
 .tt-suggestion:hover {
   cursor: pointer;
   color: #fff;
```

Comment:
If the background colour is white when hovering the text is almost not visible. Looking at the mock-up the text colour should stay the same as when not hovering.